### ISSUE TITLE
If statements inside html tags of Named Slots not working

### ISSUE THREAD

**Issue by zagro-dev** [wrabit/django-cotton#163]
I admit it's a very specific bug and definitely not urgent, but I thought I'd pass it on:

### Versions
Python==3.10
Django==4.2.13
django-cotton==1.0.9

### Initial situation
Nested Named Slot Element:

```html
<nav class="navbar navbar-expand-lg bg-primary-subtle mb-2">
    <h1 class="navbar-brand title-5 mb-0">{{ title }}</h1>
    <button class="navbar-toggler ms-auto" ...>...</button>
    <div class="collapse navbar-collapse" id="navbarPrimaryPage">
        {% if left %}
        <div class="d-flex">
             {{ left }}
        </div>
        {% endif %}
	<ul class="navbar-nav ms-auto mb-2 mb-lg-0">
	     {{ slot }}
	</ul>
    </div>
</nav>
```

This is how I use it:

```html
<c-navbar-primary-page title="My Navbar Title">
    <c-slot name="left">
	<select class="form-select" aria-label="Default select example 1">
	    {% for i in "123" %}
	    <option {% if i == '1' %}selected=""{% endif %}>{{i}} {% if i == '1' %}(selected){% endif %}</option>
	    {% endfor %}
	</select>
    </c-slot>

    <li class="nav-item px-1">
        <select class="form-select" aria-label="Default select example 2">
            {% for i in "123" %}
            <option {% if i == '1' %}selected=""{% endif %}>{{i}} {% if i == '1' %}(selected){% endif %}</option>
             {% endfor %}
        </select>
    </li>
</c-navbar-primary-page>
```

### Expected Behaviour:

I expect both `<select>` options to be identical with only the first element selected.

**Select 1 inisde the Nested Element `name="left"`**

```html
<select aria-label="Default select example 1" class="form-select">
    <option selected="">1 (selected)</option>
    <option selected="">2 </option>
    <option selected="">3 </option>
</select>

```

**Select 2 inside the default `{{slot}}`**

```html
<select aria-label="Default select example 2" class="form-select">
    <option selected="">1 (selected)</option>
    <option>2 </option>
    <option>3 </option>
</select>

```

Inside the text itself the if-statement gets evaluated correctly and only for the 1st element `(selected)` is shown. However, inside tags the `selected=""` is always visible.

This is only true for Linux (I use CentOs 8 with gunicorn 22.0.0). On my dev windows 11 machine with PyCharm it works. So this might be an issue with any other software or plugin but I have no clue how to pinpoint that. If you cannot recreate it it might be the latter but maybe it's an issue with cotton itself.

**Comment by wrabit** [comment]
@zagro-dev thanks for reporting - fixed and released in v1.0.10

**Comment by josephetareh** [comment]
Hey. Just wanted to say that I've been experiencing this issue as well:

```
<c-button.default
    {% if profile.user ==  request.user %}disabled{% endif %}
    type="button" variant="intermediate" size="small" ecs="mt-3 mt-lg-0 ms-lg-3 semi-bold"
>Archive</c-button.default>
```

throws this error:

`Invalid block tag on line 109: 'endif', expected 'endc'. Did you forget to register or load this tag?
`
